src/java/org/apache/cassandra/io/util/SequentialWriter.java

```diff
@@ -467,7 +502,7 @@ protected TransactionalProxy txnProxy()
      */
     protected static class BufferedFileWriterMark implements DataPosition
     {
-        final long pointer;
+        public final long pointer;
```

Comment:
public field in a protected class ? who's accessing this ?

Reply:
It is accessed at https://github.com/datastax/cassandra/pull/1795/files#diff-a050d5313980fc8421dd556773d2ab499eb3cd9c902ea562865688664b87de88R193